src/explore-education-statistics-admin/src/services/releaseVersionService.ts

```diff
@@ -169,7 +169,8 @@ export type ReleaseVersionChecklistWarning =
         | 'NoNextReleaseDate'
         | 'NoDataFiles'
         | 'NoFeaturedTables'
-        | 'NoPublicPreReleaseAccessList';
+        | 'NoPublicPreReleaseAccessList'
+        | 'UnresolvedComments';
```

Comment:
I think there's a test `renders correctly with warnings` which `UnresolvedComments` should be added to.